src/Parser/AParser.hpp

```diff
@@ -0,0 +1,82 @@
+/*
+** EPITECH PROJECT, 2022
+** plaza
+** File description:
+** AParser
+*/
+
+#ifndef APARSER_HPP_
+#define APARSER_HPP_
+
+#include "IParser.hpp"
+
+/**
+ * @brief The AParser inherit of IParser class.
```

Comment:
Documentation is not about what it IS, but rather what it DOES or even REPRESENTS